mh/pat_item.py

```diff
@@ -671,6 +673,73 @@ class LayerBinaryInfo(PatData):
         (0x03, "hunter_name")
     )
 
+class UserBinarySet(PatData):
```

Comment:
`PatData` structure are a special kind of way structure that the client use to transmit data.

A few examples can be seen:
 - 80411818 (MessageInfo)
 - 804114b0 (CircleInfo)
 - 80411df8 (FriendData)
 - 80410e60 (LayerData)

If you take a look at those function you see how the data is read and see how `PatData` is being serialized. You would realize that making `UserBinarySet` a `PatData` doesn't make sense.